src/blocks/net.rs

```diff
@@ -543,12 +545,25 @@ impl Net {
 
         self.speed_down = rx_bytes as f64;
 
-        self.rx_buff.remove(0);
-        self.rx_buff.push(rx_bytes as f64);
-        self.graph_rx = format_vec_to_bar_graph(&self.rx_buff, None, None);
+        self.rx_buff.pop_front();
+        self.rx_buff.push_back(rx_bytes as f64);
 
         Ok(())
     }
+
+    fn max_min_width(format: (Option<&Placeholder>, Option<&Placeholder>)) -> Option<usize> {
+        let format = match format {
+            (Some(full), Some(short)) => vec![full, short],
+            (Some(full), None) => vec![full],
+            (None, Some(short)) => vec![short],
+            (None, None) => vec![],
```

Comment:
I haven't checked if `rustc` optimizes those `vec`s out, but maybe something like this would be better?

```rust
    fn max_min_width(format: (Option<&Placeholder>, Option<&Placeholder>)) -> Option<usize> {
        match format {
            (Some(full), Some(short)) => {
                let (a, b) = (full.min_width.value, short.min_width.value);
                Some(a.or(b)?.max(b.or(a)?))
            }
            (Some(f), None) | (None, Some(f)) => f.min_width.value,
            (None, None) => None,
        }
    }
```